### PR TITLE
Universal sorted view

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**
Sorted view to support both inclusive and exclusive modes at the same time.
Removed get_quantiles() method since sorted view is cached in a sketch now, and also can be obtained and queried many times.
More consistent terminology using a generic term "item"

### CODE REVIEW

File: python/src/vector_of_kll.cpp [hunk 1]
```diff
@@ -276,31 +272,31 @@ py::array vector_of_kll_sketches<T,C,S>::get_num_retained() const {
 
 // Gets the minimum value of each sketch
 // TODO: allow subsets of sketches
-template<typename T, typename C, typename S>
-py::array vector_of_kll_sketches<T,C,S>::get_min_values() const {
+template<typename T, typename C>
+py::array vector_of_kll_sketches<T, C>::get_min_values() const {
```
Comment (jmalkin) [inline python/src/vector_of_kll.cpp:276]: do we want to stick with values() here or switch to items()? we can probably make the sketches hold py::objects eventually.

Comment (AlexanderSaydakov) [reply python/src/vector_of_kll.cpp:276]: missed this. will fix

Comment (AlexanderSaydakov) [reply python/src/vector_of_kll.cpp:276]: we instantiate vector_of_kll_sketches for int and float. perhaps we should keep "values" there

File: quantiles/include/quantiles_sketch.hpp [hunk 2]
```diff
@@ -549,7 +530,7 @@ class quantiles_sketch {
   static uint32_t compute_retained_items(uint16_t k, uint64_t n);
   static uint32_t compute_base_buffer_items(uint16_t k, uint64_t n);
   static uint64_t compute_bit_pattern(uint16_t k, uint64_t n);
-  static uint32_t compute_valid_levels(uint64_t bit_pattern);
+  static uint32_t count_valid_levels(uint64_t bit_pattern);
```
Comment (jmalkin) [inline quantiles/include/quantiles_sketch.hpp:533]: not sure why the change here?

File: quantiles/include/quantiles_sketch.hpp [hunk 3]
```diff
@@ -504,21 +481,25 @@ class quantiles_sketch {
   static const size_t DATA_START = 16;
 
   Allocator allocator_;
+  bool is_level_zero_sorted_;
```
Comment (jmalkin) [inline quantiles/include/quantiles_sketch.hpp:484]: if renaming this, it needs to be `is_base_buffer_sorted_` -- it is not levels_[0] but a different structure.

**Review by jmalkin — CHANGES_REQUESTED** [review]
i think we need to rename `is_level_zero_sorted_` in the classic quantiles sketch to `is_base_buffer_sorted_` since level zero refers to a different vector.

File: quantiles/include/quantiles_sketch.hpp (hunk 2 above)
Comment (AlexanderSaydakov) [reply quantiles/include/quantiles_sketch.hpp:533]: I felt that "compute" was too generic and vague. we really are counting them (counting set bits in the bit pattern)

Comment (jmalkin) [reply quantiles/include/quantiles_sketch.hpp:533]: everything else is `compute` but that's ok

**Review by jmalkin — APPROVED** [review]
👍
